### PR TITLE
Documentation adaptions

### PR DESCRIPTION
**Pull request by FinalAngel**
- Make installation and running of docs simpler
- Adds release policies for #4341
- Adds documentation for page types #3625

### CODE REVIEW

**Comment by evildmp** [comment]
We have quite a mix of ``build`` and ``_build`` now. Any reason not to stick with ``build``?

**Comment by FinalAngel** [comment]
@evildmp the default from sphinx is also ``_build``: https://github.com/sphinx-doc/sphinx/blob/470bac3d1c17f2728573f96c2864ab07918692d2/sphinx/quickstart.py#L1167

**Comment by evildmp** [comment]
Personally I prefer ``build``, since that is what we have been using. If we do change it to ``_build``, there will be many instances where it needs to be updated, such as in ``https://github.com/divio/django-cms/blob/develop/docs/Makefile``.

File: docs/how_to/page_types.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+pages from predefined presets. This presets contain example plugins and content
+that will be carried over the the designated page while leaving the preset
+untouched.
+
+
+*******************
+Creating Page Types
+*******************
+
+First you need to create a new page through **Add page** from the **Page**
```
Comment (evildmp) [inline docs/how_to/page_types.rst]: Only buttons should be **bold**; menu items should be *italic* (fyi, fixing here anyway).

Comment (FinalAngel) [inline docs/how_to/page_types.rst]: thanks for clarifying @evildmp

**Comment by evildmp** [comment]
Let's merge. All looks good to me.

**Pull request merged**